README.md

```diff
@@ -169,9 +174,9 @@ To interact with a CloudFoundry environment you will need the following:
 
 - the `cf` command line tool ([installation instructions](https://github.com/cloudfoundry/cli#downloads))
 - `API_ENDPOINT` from `make dev showenv`
-- `uaa_admin_password` from `cf-secrets.yml` in the state bucket
+- `uaa_admin_password` from `cf-secrets.yml` in the state bucket (should be at `aws s3 ls "s3://${DEPLOY_ENV}-state/cf-secrets.yml"`)
```

Comment:
And if you're doing that you could replace the command with `aws s3 cp "s3://${DEPLOY_ENV}-state/cf-secrets.yml" - | grep uaa_admin_password` to fetch the password directly.